### DISCUSSION TITLE
v4.5.7

### DISCUSSION THREAD

**Discussion by anhoder** [go-musicfox/go-musicfox#328] [General]
### Bug fixes
* 1629dd8 fix: panic(index out of ran) #327
### Others
* fa7b63a chore: update scoop config & changelog

<hr /><em>This discussion was created from the release <a href='https://github.com/go-musicfox/go-musicfox/releases/tag/v4.5.7'>v4.5.7</a>.</em>

**Comment by sanbei101** [comment]
so amazing!
